.github/workflows/ci.yml

```diff
@@ -331,6 +331,20 @@ jobs:
       with:
         file: /home/runner/build/coverage.info
 
+  misspell:
+    runs-on: ubuntu-latest
+    steps:
+    - name: check out code
+      uses: actions/checkout@v2
+    - name: install misspell
+      run: |
+        curl -L -o ./install-misspell.sh https://git.io/misspell
+        sh ./install-misspell.sh
+
+    - name: run misspell
+      run: ./bin/misspell -error .
+
```

Comment:
nit: an extra new line, and the new line at line 343 is unnecessary.

Reply:
Nice catch. Fixed.